src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class Ball {
+
+  private String color;
+  private int number;
+
+  public String getColor() {
+    return color;
+  }
+
+  public void setColor(String color) {
+    this.color = color.toLowerCase();
```

Comment:
Converting the color to lowercase in the setter might lead to unexpected behavior if the color is expected to be case-sensitive elsewhere in the application. If you need the color to be in lowercase, consider ensuring that the input provided to this method is already in the correct case, or document this behavior clearly.